user-manual/ingest/ingest.rst

```diff
@@ -65,6 +65,41 @@ Create a SIP
 #. For selecting a normalization option, see :ref:`Normalization <normalize>`,
    below.
 
+The microservices that run on the ingest tab include:
+
+* **Verify SIP compliance**: verifies that the SIP conforms to the folder structure required by Archivematica for processing.
+
+* **Rename SIP directory with SIP UUID**: associates the SIP with its metadata by appending the SIP UUID to the SIP directory name and checks if SIP is from Maildir transfer type to determine workflow. Both the transfer UUID and the SIP UUID are captured in the final AIP.
+
+* **Normalize**: :ref:` normalization <normalize>`converts ingested digital objects to preferred preservation and/or access formats according to user preference, using rules from :ref:`Preservation planning <preservation-planning>`. Choices about normalization can be made at the time the service is run, or automated in  :ref:`Processing configuration <dashboard-processing>`.
```

Comment:
Needs a space between `<normalize>` and `converts` in the first line - I think this'll be easier to fix when you're adjusting the lines, doing it as a suggestion here could cause a conflict.